crates/astria-composer/tests/blackbox/helper/mod.rs

```diff
@@ -50,11 +51,6 @@ pub struct TestComposer {
 pub async fn spawn_composer(rollup_ids: &[&str]) -> TestComposer {
     Lazy::force(&TELEMETRY);
 
-    assert!(
```

Comment:
this can happen if we are not running geth collectors but we are running a gRPC collector. so there could be a case where rollup_ids is empty. I can revert the code but the tests will look a bit clunky.

Reply:
reverted